packages/nx/src/native/native-file-cache-location.ts

```diff
@@ -0,0 +1,10 @@
+import { tmpdir } from 'os';
+import { join } from 'path';
+import { createHash } from 'crypto';
+import { workspaceRoot } from '../utils/workspace-root';
+
+export const nativeFileCacheLocation = join(
+  tmpdir(),
+  'nx-native-file-cache',
+  createHash('sha256').update(workspaceRoot).digest('hex')
```

Comment:
Why do we need this nesting at all? Can't it be `join(tmpdir(), 'nx/bin')`?

Reply:
I want to keep the same behaviour as before where the files are workspace-scoped. I'm not sure if there could be funky side-effects when one repo calls nx reset and deletes the `.node` file that is currently used by another process in another workspace.

Reply:
Okay sounds good.